CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [1.2.1] - 2/2/2022
```

Comment:
You've got 1.2.1 here, but changed the pubspec version to 1.3.0. Let's go with 1.3.0 here, since this is a new feature
```suggestion
## [1.3.0] - 2/2/2022
```